src/iplant_groups/service/format.clj

```diff
@@ -117,7 +117,7 @@
          :subject   (format-subject attribute-names (subject-key privilege))}
         (remove-vals nil?)))
   ([attribute-names privilege]
-   (format-privilege attribute-names privilege :ownerSubject)))
+   (format-privilege attribute-names privilege :wsSubject)))
```

Comment:
Using `ownerSubject` by default was causing weird results when privileges were assigned to a group. I changed the default to `wsSubject` and got rid of all explicit `wsSubject` parameters. Technically, this parameter can be removed, but it's not hurting anything.